pkgs/development/libraries/libressl/default.nix

```diff
@@ -11,6 +11,8 @@ let
       inherit sha256;
     };
 
+    configureFlags = [ "--enable-nc" ];
+
     enableParallelBuilding = true;
 
     outputs = [ "bin" "dev" "out" "man" ];
```

Comment:
Maybe a separate output for netcat? It feels weird to me that if I install netcat I also get a ssl library.

Reply:
Yes i can look into it. My hope was that it would be small enough to not matter though.

Reply:
The problem with somethig like this is that it’s not obvious what is in the netcat output. For instance should nc.1 go in man or netcat?

Reply:
Everything that belongs to netcat should also go there.